dsl/docker-multiarch/images/wordpress.groovy

```diff
@@ -0,0 +1,31 @@
+import vars.multiarch
+
+for (arch in multiarch.allArches()) {
+	meta = multiarch.meta(getClass(), arch)
+
+	freeStyleJob(meta.name) {
+		description(meta.description)
+		logRotator { daysToKeep(30) }
+		label(meta.label)
+		scm {
+			git {
+				remote { url('https://github.com/docker-library/wordpress.git') }
+				branches('*/master')
+				clean()
+			}
+		}
+		triggers {
+			upstream("docker-${arch}-php", 'UNSTABLE')
+			scm('H H/6 * * *')
+		}
+		wrappers { colorizeOutput() }
+		steps {
+			shell(multiarch.templateArgs(meta, ['dpkgArch']) + '''
+sed -i "s!^FROM !FROM $prefix/!" */{,*/}Dockerfile
```

Comment:
The arg here should just be `*/Dockerfile` -- what's here will expand to `*/Dockerfile` and `*/*/Dockerfile`, and the second of these won't match anything since Wordpress doesn't have any sub-variants. :smile: